cfy_manager/components/restservice/db.py

```diff
@@ -44,6 +48,31 @@
 REST_HOME_DIR = '/opt/manager'
 
 
+def _connect_to_db(query_func):
+    def wrapper(*args, **kwargs):
+        pg_config = config[POSTGRESQL_CLIENT]
+        db_connection_string = \
+            'postgres://{user}:{password}@{hostname_and_port}/{db}'.format(
+                user=pg_config['username'],
+                password=pg_config['password'],
+                hostname_and_port=pg_config['host'],
+                db=pg_config['db_name']
+            )
+        try:
+            connection = create_engine(db_connection_string,
+                                       poolclass=NullPool).connect()
+
+            return_value = query_func(connection=connection, *args, **kwargs)
+
+            connection.close()
+            return return_value
+        except Exception as err:
+            logger.debug(err)
```

Comment:
I think join the lines to one log msg, it will be more clear to understand to see that those two line are connected events

Reply:
Fixed